untappdFeedParser.js

```diff
@@ -25,6 +25,13 @@ const cities = process.env.CITIES.split(" ").map(value => {
 
 const timeFormat = 'ddd, DD MMM YYYY HH:mm:ss Z';
 
+const isDuringWorkTime = instant => {
+  if (instant.isoWeekday() <= 6) {
```

Comment:
Shouldn't the comparison be the other way around? I.e.

```suggestion
  if (instant.isoWeekday() >= 6) {
```

So if isoWeekday is 6/7 (Saturday/Sunday) it's not working time.